js/start.js

```diff
@@ -74,6 +76,15 @@ app.loadingModal = new LoadingModal({
   removeOnRoute: false,
 }).render().open();
 
+// fix zoom issue on Linux hiDPI
+if (platform === 'linux') {
+  let scaleFactor = screen.getPrimaryDisplay().scaleFactor;
+  if (scaleFactor === 0) {
+    scaleFactor = 1;
+  }
+  $('body').css('zoom', 1 / scaleFactor);
```

Comment:
Could we please use getBody from the utils/selectors.js file? It will cache that element so we only need to go to the DOM once for it.